src/apps/code-editor/src/store/navCode.js

```diff
@@ -248,19 +248,24 @@ export const buildNavTree = (nodes, type) => {
       const fileNameParts = node.fileName.replace(/^\//, "").split("/");
       fileNameParts.reduce((prevParts, part, i) => {
         // Are we at the last part of the filename, the "file"
+        const nodeSort = !!node.sort ? `(${node.sort}) ` : "";
```

Comment:
Same here, is this solving a different issue aside from the original issue raised?